### PR TITLE
Feature/documentation updates

### PR DESCRIPTION
**Pull request by TSteele27**

### CODE REVIEW

**Review by rajeshsusai — APPROVED** [review]

File: docs/packages/cli/README.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ new resources and dependencies,
 
 ### Start
 
+**[DEPRECATED]**
```
Comment (BReul) [inline docs/packages/cli/README.md]: this was a bit confusing to me.  Maybe put deprecated in the start header?

File: packages/cli/README.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ new resources and dependencies,
 
 ### Start
 
+**[DEPRECATED]**
```
Comment (BReul) [inline packages/cli/README.md]: same as previous comment, maybe move this into the start header

**Review by BReul — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -19,3 +19,15 @@ All contributions should follow the existing style of the package and pass our e
...
+
+
+
+## Other ways to get involved
+
+There are more ways to contribute to BMO than just new features. Bug fixes, additions to the documentation,
+new modules and extensions are just some of the other ways you can contribute. If you have an awesome extension
+or have written a new module we want to know!
+
+If you think updates to the docs are in order feel free to follow the contributions guidelines above for documentation updates.
```
Comment (srpiatt) [inline CONTRIBUTING.md]:
```suggestion
If you think updates to the docs are in order, feel free to follow the contributions guidelines above for documentation updates.
```

File: docs/packages/bundle/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+```
+
+Load a bundle:
+
+```
+await app.load(config)
+```
+
+Loads a configuration and dependency bundle, but does not resolve or perform the injection.
+config is the same as the constructor and takes precedent over the config passed there.
```
Comment (srpiatt) [inline docs/packages/bundle/README.md]:
```suggestion
The config parameter is the same as the constructor and takes precedent over the config value passed there.
```

File: docs/packages/injector/README.md [hunk 5]
```diff
@@ -51,6 +51,11 @@ const manifest = await inject(config, dependencies)
 When the manifest is created the injector traverses your dependency tree injecting each modules dependencies
 eventually ending up with an object with all your build modules.
 
+### Package dependencies and built in modules:
+The injector will detect and load nodejs builtin modules into your dependency context for you.
+To use packages defined in your package.json populate the 'pkg' key on your config with the package.json
```
Comment (srpiatt) [inline docs/packages/injector/README.md]:
```suggestion
To use packages defined in your package.json, populate the 'pkg' key on your config with the package.json
```

File: packages/run/README.md [hunk 6]
```diff
@@ -1 +1,19 @@
-README.md
+# @b-mo/extension-run
+
+This is an extension for the bmo cli, its purpose it to provide a simple interface to run your project as dependency bundle.
```
Comment (srpiatt) [inline packages/run/README.md]:
```suggestion
This is an extension for the bmo cli; its purpose it to provide a simple interface to run your project as a dependency bundle.
```

**Review by srpiatt — APPROVED** [review]
Just some misc comma stuffs that you could do, but def not required. Looks great! :D

**Pull request merged**